go.mod

```
@@ -1,56 +1,29 @@
 module go-google-scraper-challenge
 
-go 1.15
+go 1.16
```

```suggestion
go 1.18
```
Wanna try on with the latest version? 😛